validator/sawtooth_validator/gossip/permission_verifier.py

```diff
@@ -13,13 +13,16 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 import logging
-# pylint: disable=import-error,no-name-in-module
-# needed for google.protobuf import
+
+# pylint: disable=no-name-in-module,import-error
```

Comment:
It might have been a pylint change, but this isn't necessary any more. I tested it locally without the disable pylint comment and it showed no lint.